2014-05-30 14:04:14,185 [main] INFO  BaseModule.common.DebugLog - Starting to load catData: src/main/resources/CatData
2014-05-30 14:04:14,872 [main] INFO  BaseModule.common.DebugLog - catData loaded succesfully
2014-05-30 14:04:14,873 [main] INFO  BaseModule.common.DebugLog - Starting to load locationData: src/main/resources/LocationData
2014-05-30 14:04:14,873 [main] INFO  BaseModule.common.DebugLog - locationData loaded succesfully
2014-05-30 14:04:15,225 [main] INFO  BaseModule.common.DebugLog - BaseModule.service.ModelDataLoaderService got Exception! 
BaseModule.exception.validation.ValidationException
	at BaseModule.eduDAO.UserDao.addUserToDatabase(UserDao.java:102)
	at BaseModule.service.ModelDataLoaderService.loadUsers(ModelDataLoaderService.java:164)
	at BaseModule.service.ModelDataLoaderService.load(ModelDataLoaderService.java:41)
	at UserModule.serverMain.ServerMain.main(ServerMain.java:71)
2014-05-30 14:04:15,250 [main] INFO  BaseModule.common.DebugLog - BaseModule.eduDAO.PartnerDao got Exception! 
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'XDF' for key 'name_UNIQUE'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	at com.mysql.jdbc.Util.getInstance(Util.java:386)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1040)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4120)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4052)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2503)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2664)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2794)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:2155)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2458)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2375)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2359)
	at com.zaxxer.hikari.proxy.PreparedStatementJavassistProxy.executeUpdate(PreparedStatementJavassistProxy.java)
	at BaseModule.eduDAO.PartnerDao.addPartnerToDatabases(PartnerDao.java:95)
	at BaseModule.service.ModelDataLoaderService.loadPartners(ModelDataLoaderService.java:207)
	at BaseModule.service.ModelDataLoaderService.load(ModelDataLoaderService.java:42)
	at UserModule.serverMain.ServerMain.main(ServerMain.java:71)
2014-05-30 14:04:15,265 [main] INFO  BaseModule.common.DebugLog - BaseModule.eduDAO.PartnerDao got Exception! 
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'HQYS' for key 'name_UNIQUE'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	at com.mysql.jdbc.Util.getInstance(Util.java:386)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1040)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4120)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4052)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2503)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2664)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2794)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:2155)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2458)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2375)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2359)
	at com.zaxxer.hikari.proxy.PreparedStatementJavassistProxy.executeUpdate(PreparedStatementJavassistProxy.java)
	at BaseModule.eduDAO.PartnerDao.addPartnerToDatabases(PartnerDao.java:95)
	at BaseModule.service.ModelDataLoaderService.loadPartners(ModelDataLoaderService.java:218)
	at BaseModule.service.ModelDataLoaderService.load(ModelDataLoaderService.java:42)
	at UserModule.serverMain.ServerMain.main(ServerMain.java:71)
2014-05-30 14:04:15,281 [main] INFO  BaseModule.common.DebugLog - BaseModule.eduDAO.PartnerDao got Exception! 
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'XDFs' for key 'name_UNIQUE'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	at com.mysql.jdbc.Util.getInstance(Util.java:386)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1040)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4120)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4052)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2503)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2664)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2794)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:2155)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2458)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2375)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2359)
	at com.zaxxer.hikari.proxy.PreparedStatementJavassistProxy.executeUpdate(PreparedStatementJavassistProxy.java)
	at BaseModule.eduDAO.PartnerDao.addPartnerToDatabases(PartnerDao.java:95)
	at BaseModule.service.ModelDataLoaderService.loadPartners(ModelDataLoaderService.java:229)
	at BaseModule.service.ModelDataLoaderService.load(ModelDataLoaderService.java:42)
	at UserModule.serverMain.ServerMain.main(ServerMain.java:71)
2014-05-30 14:04:15,298 [main] INFO  BaseModule.common.DebugLog - BaseModule.eduDAO.AdminAccountDao got Exception! 
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '123445676543' for key 'phone_UNIQUE'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	at com.mysql.jdbc.Util.getInstance(Util.java:386)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1040)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4120)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4052)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2503)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2664)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2794)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:2155)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2458)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2375)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2359)
	at com.zaxxer.hikari.proxy.PreparedStatementJavassistProxy.executeUpdate(PreparedStatementJavassistProxy.java)
	at BaseModule.eduDAO.AdminAccountDao.addAdminAccountToDatabases(AdminAccountDao.java:85)
	at BaseModule.service.ModelDataLoaderService.loadAdmins(ModelDataLoaderService.java:245)
	at BaseModule.service.ModelDataLoaderService.load(ModelDataLoaderService.java:43)
	at UserModule.serverMain.ServerMain.main(ServerMain.java:71)
2014-05-30 14:04:15,312 [main] INFO  BaseModule.common.DebugLog - BaseModule.eduDAO.AdminAccountDao got Exception! 
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '12344' for key 'phone_UNIQUE'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	at com.mysql.jdbc.Util.getInstance(Util.java:386)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1040)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4120)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4052)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2503)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2664)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2794)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:2155)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2458)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2375)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2359)
	at com.zaxxer.hikari.proxy.PreparedStatementJavassistProxy.executeUpdate(PreparedStatementJavassistProxy.java)
	at BaseModule.eduDAO.AdminAccountDao.addAdminAccountToDatabases(AdminAccountDao.java:85)
	at BaseModule.service.ModelDataLoaderService.loadAdmins(ModelDataLoaderService.java:253)
	at BaseModule.service.ModelDataLoaderService.load(ModelDataLoaderService.java:43)
	at UserModule.serverMain.ServerMain.main(ServerMain.java:71)
2014-05-30 14:04:15,326 [main] INFO  BaseModule.common.DebugLog - BaseModule.eduDAO.AdminAccountDao got Exception! 
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '2323' for key 'phone_UNIQUE'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	at com.mysql.jdbc.Util.getInstance(Util.java:386)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1040)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4120)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4052)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2503)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2664)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2794)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:2155)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2458)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2375)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2359)
	at com.zaxxer.hikari.proxy.PreparedStatementJavassistProxy.executeUpdate(PreparedStatementJavassistProxy.java)
	at BaseModule.eduDAO.AdminAccountDao.addAdminAccountToDatabases(AdminAccountDao.java:85)
	at BaseModule.service.ModelDataLoaderService.loadAdmins(ModelDataLoaderService.java:261)
	at BaseModule.service.ModelDataLoaderService.load(ModelDataLoaderService.java:43)
	at UserModule.serverMain.ServerMain.main(ServerMain.java:71)
2014-05-30 14:04:15,824 [main] INFO  BaseModule.common.DebugLog - ready to start
2014-05-30 14:04:15,909 [main] INFO  BaseModule.common.DebugLog - initiaing router::RoutingService
2014-05-30 14:04:15,921 [main] INFO  BaseModule.common.DebugLog - Excuting
2014-05-30 14:05:19,629 [main] INFO  BaseModule.common.DebugLog - Starting to load catData: src/main/resources/CatData
2014-05-30 14:05:20,296 [main] INFO  BaseModule.common.DebugLog - catData loaded succesfully
2014-05-30 14:05:20,296 [main] INFO  BaseModule.common.DebugLog - Starting to load locationData: src/main/resources/LocationData
2014-05-30 14:05:20,297 [main] INFO  BaseModule.common.DebugLog - locationData loaded succesfully
2014-05-30 14:05:21,403 [main] INFO  BaseModule.common.DebugLog - ready to start
2014-05-30 14:05:21,430 [main] INFO  BaseModule.common.DebugLog - initiaing router::RoutingService
2014-05-30 14:05:21,438 [main] INFO  BaseModule.common.DebugLog - Excuting
2014-05-30 14:05:36,929 [Restlet-1185074162] INFO  BaseModule.common.DebugLog - BaseModule.resources.PseudoResource got Exception! 
BaseModule.exception.AuthenticationException
	at UserModule.service.UserAuthenticationService.validateSession(UserAuthenticationService.java:13)
	at UserModule.resources.UserPseudoResource.validateAuthentication(UserPseudoResource.java:29)
	at UserModule.resources.user.UserIdResource.getUserById(UserIdResource.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.restlet.resource.ServerResource.doHandle(ServerResource.java:506)
	at org.restlet.resource.ServerResource.get(ServerResource.java:707)
	at org.restlet.resource.ServerResource.doHandle(ServerResource.java:589)
	at org.restlet.resource.ServerResource.doNegotiatedHandle(ServerResource.java:649)
	at org.restlet.resource.ServerResource.doConditionalHandle(ServerResource.java:348)
	at org.restlet.resource.ServerResource.handle(ServerResource.java:952)
	at org.restlet.resource.Finder.handle(Finder.java:246)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Router.doHandle(Router.java:431)
	at org.restlet.routing.Router.handle(Router.java:648)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.engine.application.StatusFilter.doHandle(StatusFilter.java:155)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.engine.CompositeHelper.handle(CompositeHelper.java:211)
	at org.restlet.engine.application.ApplicationHelper.handle(ApplicationHelper.java:84)
	at org.restlet.Application.handle(Application.java:381)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Router.doHandle(Router.java:431)
	at org.restlet.routing.Router.handle(Router.java:648)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Router.doHandle(Router.java:431)
	at org.restlet.routing.Router.handle(Router.java:648)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.engine.application.StatusFilter.doHandle(StatusFilter.java:155)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.engine.CompositeHelper.handle(CompositeHelper.java:211)
	at org.restlet.Component.handle(Component.java:392)
	at org.restlet.Server.handle(Server.java:516)
	at org.restlet.engine.connector.ServerConnectionHelper.handle(ServerConnectionHelper.java:257)
	at org.restlet.engine.connector.ServerConnectionHelper.doHandleInbound(ServerConnectionHelper.java:186)
	at org.restlet.engine.connector.BaseHelper$2.run(BaseHelper.java:593)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2014-05-30 14:05:46,854 [Restlet-1185074162] INFO  BaseModule.common.DebugLog - BaseModule.resources.PseudoResource got Exception! 
BaseModule.exception.AuthenticationException
	at UserModule.service.UserAuthenticationService.validateSession(UserAuthenticationService.java:13)
	at UserModule.resources.UserPseudoResource.validateAuthentication(UserPseudoResource.java:29)
	at UserModule.resources.user.UserResource.getAllUsers(UserResource.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.restlet.resource.ServerResource.doHandle(ServerResource.java:506)
	at org.restlet.resource.ServerResource.get(ServerResource.java:707)
	at org.restlet.resource.ServerResource.doHandle(ServerResource.java:589)
	at org.restlet.resource.ServerResource.doNegotiatedHandle(ServerResource.java:649)
	at org.restlet.resource.ServerResource.doConditionalHandle(ServerResource.java:348)
	at org.restlet.resource.ServerResource.handle(ServerResource.java:952)
	at org.restlet.resource.Finder.handle(Finder.java:246)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Router.doHandle(Router.java:431)
	at org.restlet.routing.Router.handle(Router.java:648)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.engine.application.StatusFilter.doHandle(StatusFilter.java:155)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.engine.CompositeHelper.handle(CompositeHelper.java:211)
	at org.restlet.engine.application.ApplicationHelper.handle(ApplicationHelper.java:84)
	at org.restlet.Application.handle(Application.java:381)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Router.doHandle(Router.java:431)
	at org.restlet.routing.Router.handle(Router.java:648)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Router.doHandle(Router.java:431)
	at org.restlet.routing.Router.handle(Router.java:648)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.engine.application.StatusFilter.doHandle(StatusFilter.java:155)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.engine.CompositeHelper.handle(CompositeHelper.java:211)
	at org.restlet.Component.handle(Component.java:392)
	at org.restlet.Server.handle(Server.java:516)
	at org.restlet.engine.connector.ServerConnectionHelper.handle(ServerConnectionHelper.java:257)
	at org.restlet.engine.connector.ServerConnectionHelper.doHandleInbound(ServerConnectionHelper.java:186)
	at org.restlet.engine.connector.BaseHelper$2.run(BaseHelper.java:593)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
